nserror-kt/src/nativeInterop/cinterop/NSErrorKtRuntime.def

```diff
@@ -1,12 +1,54 @@
 ---
+
+#include "stdint.h"
+#include "stddef.h"
+
+// Based on Kotlin Native runtime
+
+typedef struct ObjHeader ObjHeader;
+
+typedef struct FrameOverlay {
+  struct FrameOverlay* previous;
+  int32_t parameters;
+  int32_t count;
+} FrameOverlay;
+
+typedef struct ObjHolder {
+    FrameOverlay frame_;
+    ObjHeader* obj_;
+} ObjHolder;
+
+extern void EnterFrame(ObjHeader** start, int parameters, int count) __attribute__((nothrow));
+extern void LeaveFrame(ObjHeader** start, int parameters, int count) __attribute__((nothrow));
+
+static void NSErrorKt_Kotlin_ObjCExport_ObjHolder_init(ObjHolder* this) {
```

Comment:
I would prefer to remove `Kotlin_ObjCExport` from the name to distinguish between custom functions and simple wrappers.

Reply:
Yeah, why not. The function is static, so there is no need to have a unique name.